packages/core/src/ui/UIImage.ts

```
@@ -0,0 +1,293 @@
+    this._sprite = null;
+    this._assembler = null;
+  }
+
+  @ignoreClone
+  private _onSpriteChange(type: SpriteModifyFlags): void {
+    switch (type) {
+      case SpriteModifyFlags.texture:
+        this.shaderData.setTexture(UIRenderer._textureProperty, this.sprite.texture);
+        break;
+      case SpriteModifyFlags.size:
+        const { _drawMode: drawMode } = this;
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)